Cloudflare/audit_logs/ingest/parser.yml

```
@@ -0,0 +1,23 @@
+name: cloudflare-audit-logs
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.When}}'
+          user.email: '{{json_event.message.ActorEmail}}'
+          user.id: '{{json_event.message.ActorID}}'
+          source.ip: '{{json_event.message.ActorIP}}'
```

I suggest adding the following fields
```suggestion
          source.ip: '{{json_event.message.ActorIP}}'
          event.category: ["network"]
          event.dataset: "audit_logs"
          event.kind: 'event'
          event.type: ["access"]
          observer.type: "proxy"
          observer.vendor: "Cloudflare"
```